### PR TITLE
Add epoch start and epoch end to activities

### PR DESCRIPTION
**Pull request by crabsinger**
- Implement activity for epoch start and end
- Create UI for epoch started and ended

image

### CODE REVIEW

File: src/features/activities/EpochEndedRow.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        borderBottom: '1px solid $dim',
+        py: '$md',
+        alignItems: 'center',
+      }}
+    >
+      <Epoch size="xl" nostroke css={{ mr: '$lg', ml: '$sm' }} />
+      <Flex column>
+        <Flex>
+          <Text inline color="cta" size="small">
+            Epoch {activity.epoch.number} Ended
```
[the remark this answers is not in the document]
Comment (CryptoGraffe) [reply src/features/activities/EpochEndedRow.tsx:22]: lets do a follow-on w/ content/style improvements

**Review by CryptoGraffe — CHANGES_REQUESTED** [review]

**Comment by CryptoGraffe** [comment]
@crabsinger reviewed with some minor notes. there is also a failing test

File: api/hasura/cron/epochs.ts [hunk 2]
```diff
@@ -272,7 +279,6 @@ export async function notifyEpochStart({
         },
         sanitize: false,
       });
-      updateEpochStartNotification(epoch.id);
```
Comment (crabsinger) [inline api/hasura/cron/epochs.ts:275]: rather than doing this after reach notiifcation, I'm just doing it always at the bottom with the start activity.

**Comment by crabsinger** [comment]
@CryptoGraffe ready for re-review - yes, UI can show more info, lets pair on it later

**Pull request merged**